src/backend/joanie/tests/payment/test_models_credit_card.py

```diff
@@ -0,0 +1,37 @@
+"""Test suite for the `CreditCard` Manager."""
+
+import uuid
+
+from django.test import TestCase
+
+from joanie.core.factories import UserFactory
+from joanie.payment.factories import CreditCardFactory
+from joanie.payment.models import CreditCard
+
+
+class CreditCardModelTestCase(TestCase):
+    """
+    Test case for the CreditCard Manager.
+    """
+
+    def test_models_credit_card_get_card_for_owner(self):
+        """
+        If the `pk` and the `owner_id` matches an existing credit card, the manager
+        method `get_card_for_owner` of the `CreditCard` model should return the object.
+        When it does not match any objects in the database, it should return `None`.
+        """
+        user = UserFactory()
+        credit_card = CreditCardFactory(owner=user, is_main=True)
+
+        credit_card = CreditCard.objects.get_card_for_owner(
+            pk=credit_card.pk, owner_id=user.id
+        )
+
+        self.assertEqual(credit_card.owner, user)
+
+        with self.assertRaises(CreditCard.DoesNotExist) as context:
+            CreditCard.objects.get_card_for_owner(pk=uuid.uuid4(), owner_id=user.id)
```

Comment:
IMO this test is not relevant. You know the randon `uuid` generated is not in the database. You should create a real credit card linked to an other user.